### ISSUE TITLE
Failing MOI's indicator4 test

### ISSUE THREAD

**Issue by joaquimg** [SCIP-Interfaces/SCIP.jl#147]
Julia version: 1.3.1 (windows)
SCIP version: 0.9.1
MOI version: 0.9.12

```
indicator4: Test Failed at C:\Users\joaquimgarcia\.julia\packages\MathOptInterface\XiH8D\src\Test\intlinear.jl:676
  Expression: ≈(MOI.get(model, MOI.ObjectiveValue()), 28.75, atol=atol, rtol=rtol)
   Evaluated: 30.0 ≈ 28.75 (atol=1.0e-5, rtol=1.0e-5)
Stacktrace:
 [1] indicator4_test(::MathOptInterface.Bridges.LazyBridgeOptimizer{SCIP.Optimizer}, ::MathOptInterface.Test.TestConfig{Float64}) at C:\Users\joaquimgarcia\.julia\packages\MathOptInterface\XiH8D\src\Test\intlinear.jl:676
 [2] macro expansion at C:\Users\joaquimgarcia\.julia\packages\MathOptInterface\XiH8D\src\Test\config.jl:48 [inlined]
 [3] intlineartest(::MathOptInterface.Bridges.LazyBridgeOptimizer{SCIP.Optimizer}, ::MathOptInterface.Test.TestConfig{Float64}, ::Array{String,1}) at C:\Users\joaquimgarcia\.julia\packages\MathOptInterface\XiH8D\src\Test\config.jl:48
 [4] top-level scope at C:\Users\joaquimgarcia\.julia\packages\SCIP\hrirX\test\MOI_wrapper_bridged.jl:47
 [5] top-level scope at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\Test\src\Test.jl:1107
 [6] top-level scope at C:\Users\joaquimgarcia\.julia\packages\SCIP\hrirX\test\MOI_wrapper_bridged.jl:46
indicator4: Test Failed at C:\Users\joaquimgarcia\.julia\packages\MathOptInterface\XiH8D\src\Test\intlinear.jl:677
  Expression: ≈(MOI.get(model, MOI.VariablePrimal(), x1), 1.25, atol=atol, rtol=rtol)
   Evaluated: 0.0 ≈ 1.25 (atol=1.0e-5, rtol=1.0e-5)
Stacktrace:
 [1] indicator4_test(::MathOptInterface.Bridges.LazyBridgeOptimizer{SCIP.Optimizer}, ::MathOptInterface.Test.TestConfig{Float64}) at C:\Users\joaquimgarcia\.julia\packages\MathOptInterface\XiH8D\src\Test\intlinear.jl:677
 [2] macro expansion at C:\Users\joaquimgarcia\.julia\packages\MathOptInterface\XiH8D\src\Test\config.jl:48 [inlined]
 [3] intlineartest(::MathOptInterface.Bridges.LazyBridgeOptimizer{SCIP.Optimizer}, ::MathOptInterface.Test.TestConfig{Float64}, ::Array{String,1}) at C:\Users\joaquimgarcia\.julia\packages\MathOptInterface\XiH8D\src\Test\config.jl:48
 [4] top-level scope at C:\Users\joaquimgarcia\.julia\packages\SCIP\hrirX\test\MOI_wrapper_bridged.jl:47
 [5] top-level scope at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\Test\src\Test.jl:1107
 [6] top-level scope at C:\Users\joaquimgarcia\.julia\packages\SCIP\hrirX\test\MOI_wrapper_bridged.jl:46
indicator4: Test Failed at C:\Users\joaquimgarcia\.julia\packages\MathOptInterface\XiH8D\src\Test\intlinear.jl:678
  Expression: ≈(MOI.get(model, MOI.VariablePrimal(), x2), 8.75, atol=atol, rtol=rtol)
   Evaluated: 10.0 ≈ 8.75 (atol=1.0e-5, rtol=1.0e-5)
Stacktrace:
 [1] indicator4_test(::MathOptInterface.Bridges.LazyBridgeOptimizer{SCIP.Optimizer}, ::MathOptInterface.Test.TestConfig{Float64}) at C:\Users\joaquimgarcia\.julia\packages\MathOptInterface\XiH8D\src\Test\intlinear.jl:678
 [2] macro expansion at C:\Users\joaquimgarcia\.julia\packages\MathOptInterface\XiH8D\src\Test\config.jl:48 [inlined]
 [3] intlineartest(::MathOptInterface.Bridges.LazyBridgeOptimizer{SCIP.Optimizer}, ::MathOptInterface.Test.TestConfig{Float64}, ::Array{String,1}) at C:\Users\joaquimgarcia\.julia\packages\MathOptInterface\XiH8D\src\Test\config.jl:48
 [4] top-level scope at C:\Users\joaquimgarcia\.julia\packages\SCIP\hrirX\test\MOI_wrapper_bridged.jl:47
 [5] top-level scope at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\Test\src\Test.jl:1107
 [6] top-level scope at C:\Users\joaquimgarcia\.julia\packages\SCIP\hrirX\test\MOI_wrapper_bridged.jl:46
```

**Comment by rschwarz** [comment]
Thanks for the report, I can reproduce with Julia 1.3.0 and Linux.

**Comment by rschwarz** [comment]
Interestingly, that same `indicator4_test` passes when using SCIP in direct mode.

**Comment by rschwarz** [comment]
So, this was already fixed on master, and 0.9.3 should be released soon.

**Comment by joaquimg** [comment]
Thanks!
